consensus-types/blocks/getters_test.go

```diff
@@ -312,3 +338,29 @@ func Test_BeaconBlockBody_HashTreeRoot(t *testing.T) {
 	require.NoError(t, err)
 	assert.DeepEqual(t, expectedHTR, actualHTR)
 }
+
+func hydrateSignedBeaconBlock() *eth.SignedBeaconBlock {
```

Comment:
I can't use `utils.HydrateXXX` functions here because that will result in a cyclic dependency. The `utils` package uses functions from this package.